providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/model/FlagParser.java

```diff
@@ -0,0 +1,81 @@
+package dev.openfeature.contrib.providers.flagd.resolver.process.model;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+/**
+ * flagd feature flag configuration parser.
+ */
+public class FlagParser {
```

Comment:
have you checked if we use the schema available here? https://github.com/open-feature/playground/blob/main/schemas/flag.schema.json

Reply:
We now use networknt json-schema-validator and validate the payload before accepting